File: Php/app/Homegame/Listing/HomegameItemModel.php
namespace app\Homegame\Listing{

	use entities\Homegame;
	use app\Urls\HomegameDetailsUrlModel;

	class HomegameItemModel{

		public $name;
		public $urlModel;

		public function __construct(Homegame $homegame){
			name = $homegame.getDisplayName();
			urlModel = new HomegameDetailsUrlModel($homegame);
		}

	}

}

File: Web/Models/HomegameModels/Listing/HomegameItemModel.cs
using Core.Classes;
using Web.Models.Url;

namespace Web.Models.HomegameModels.Listing{

	public class HomegameItemModel{

	    public string Name { get; set; }
	    public UrlModel UrlModel { get; set; }

	    public HomegameItemModel(Homegame homegame)
	    {
	        Name = homegame.DisplayName;
			UrlModel = new HomegameDetailsUrlModel(homegame);
	    }
        
	}

}